website/docs/extensions/write/index.md

```diff
@@ -0,0 +1,135 @@
+---
+sidebar_position: 1
+title: Writing
+description: Writing my first extension
+tags: [podman-desktop, extension, writing]
+keywords: [podman desktop, extension, writing]
+---
+
+# Writing your first extension
+
+Podman Desktop extensions rely on `JavaScript` or `TypeScript`.
+
+Extensions are self-contained, all runtime dependencies are inside the final binary.
+
+#### Scaffold the project
+
+1. Setup a Node.js project with `TypeScript`. Use for example `npm init` command.
+
+1. Bring the development dependency to Podman Desktop API. It allows to import the API and register the contributions.
+
+  Note that it is a dependency that brings only the definition of the API. It will not fetch any Podman Desktop internals.
+
+  ```json
```

Comment:
Can you clarify which file we are editing here?

Reply:
`package.json` file